nautobot/core/wsgi.py

```diff
@@ -1,11 +1,18 @@
 import logging
-import os
 
 from django.core import cache
 from django.core.wsgi import get_wsgi_application
 from django.db import connections
 
-os.environ["DJANGO_SETTINGS_MODULE"] = "nautobot_config"
+import nautobot
+
+# This is the Django default left here for visibility on how the Nautobot pattern
+# differs.
+# os.environ.setdefault("DJANGO_SETTINGS_MODULE", "nautobot.core.settings")
+
+# Instead of importing `DJANGO_SETTINGS_MODULE` we're using the custom loader
+# pattern from `nautobot.core.runner` to read environment or config path for us.
```

Comment:
```suggestion
# Instead of just pointing to `DJANGO_SETTINGS_MODULE` and letting Django run with it,
# we're using the custom Nautobot loader code to read environment or config path for us.
```

Since `nautobot.core.runner` doesn't exist any more :)